Cargo.toml

```diff
@@ -18,7 +18,7 @@ backtrace = ["failure/backtrace", "failure/std"]
 arrayref = "0.3"
 bincode = "1.0"
 lazy_static = "1.0"
-lmdb-rkv = "0.9"
+lmdb-rkv = { path = "../lmdb-rs" }
```

Comment:
Did you wind up needing to make any changes to _lmdb-rs_ in order to support MultiReader/Writer?

Reply:
Yes.  A terribly minor change: 
In https://github.com/mozilla/lmdb-rs/blob/master/src/cursor.rs#L83  I changed the key param from `&K` to `K` .  The reference seemed redundant, and this way it matches the `iter` and `iter_from` functions.